packages/atclient/CMakeLists.txt

```diff
@@ -139,18 +136,36 @@ if(NOT ESP_PLATFORM)
   else()
     message(STATUS "[ATCLIENT] MbedTLS found package..")
   endif()
-
-  if(ATCLIENT_MAKE_AVAILABLE)
-    FetchContent_MakeAvailable(${ATCLIENT_MAKE_AVAILABLE})
-  endif()
-
+  
   # # ########################################################
   # 5C. cJSON
   # # ########################################################
-  add_subdirectory(${CJSON_DIR})
-  list(APPEND ATCLIENT_TARGETS_TO_INSTALL cjson-static)
+  find_package(cjson QUIET)
+  
+  if(NOT cjson_FOUND)
+    message(STATUS "[ATCLIENT] cJSON not found, fetching from GitHub..")
+    FetchContent_Declare(
+      cjson
+      URL https://github.com/DaveGamble/cJSON/archive/refs/tags/v1.7.17.zip
+      URL_HASH SHA256=51f3b07aece8d1786e74b951fd92556506586cb36670741b6bfb79bf5d484216 # hash for v1.7.17 .zip release source code
+    )
 
-  # # ########################################################
+    list(APPEND ATCLIENT_TARGETS_TO_INSTALL cjson) # since we've fetched it, we have to install it
+    set(CJSON_BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
+    set(CJSON_OVERRIDE_BUILD_SHARED_LIBS ON CACHE BOOL "" FORCE)
+    set(ENABLE_CJSON_TEST OFF CACHE BOOL "" FORCE)
+    add_compile_options(-fPIC)
+    FetchContent_MakeAvailable(cJSON)
```

Comment:
this should be lower 'cjson' now right?

Reply:
'cjson' is the correct package name

Reply:
Thanks, good catch https://github.com/atsign-foundation/at_c/pull/181/commits/eedece0d9339bdf397421108b3801343a5c849d0